Scribe/AppTexts/AppTextStyling.swift

```diff
@@ -104,3 +177,41 @@ func setPrivacyPolicy(fontSize: CGFloat, text: String) -> NSMutableAttributedStr
 
   return privacyPolicyTextWithLinks
 }
+
+/// Formats and returns the text of the Scribe third-party licenses with links activated and list indents.
+func setThirdPartyLicenses(fontSize: CGFloat, text: String, listElements: [String]) -> NSMutableAttributedString {
+  let licensesLink = "https://github.com/EthanSK/CustomKeyboard/blob/master/LICENSE"
+  let thirdPartyLicensesTextWithLink = addHyperLinks(
+    originalText: text,
+    links: [
+      licensesLink : licensesLink
+    ],
+    fontSize: fontSize
+  )
+  
+  let thirdPartyLicensesTextWithLinkAndIndents = addTabStops(attributedOriginalText: thirdPartyLicensesTextWithLink, links: listElements)
+  
+  return thirdPartyLicensesTextWithLinkAndIndents
+}
+
+/// Formats and returns the text of the Wikimedia and Scribe screen with images.
+func setWikimediaAndScribe(text: [String], fontSize: CGFloat, imageWidth: CGFloat) -> NSMutableAttributedString {
+  var attributedTextBySections = [NSMutableAttributedString]()
+  
+  for section in text {
+    let attributedSection = NSMutableAttributedString(string: section, attributes: [NSAttributedString.Key.font: UIFont.systemFont(ofSize: fontSize)])
+    attributedTextBySections.append(attributedSection)
+  }
+  
+  let wikidataLogoString = getCenteredImagesForWikimediaAndScribe(imageString: "wikidata.logo", imageWidth: imageWidth)
+  
+  let wikipediaLogoString = getCenteredImagesForWikimediaAndScribe(imageString: "wikipedia.logo", imageWidth: imageWidth)
```

Comment:
More my curiosity - I assume that yes, but we've checked the legal stuff for using Wikimedia logos/marks (question more for @andrewtavis perhaps)? My understanding is that this would fall under this [section](https://foundation.wikimedia.org/wiki/Policy:Trademark_policy#3.6_Refer_to_Wikimedia_sites_(nominative_use))? I haven't read the entire document, but was curious on what Scribe has thought through as far as properly including Wikimedia logos/marks.

Reply:
This was something that was discussed as early as [Data Reuse Days 2022](https://diff.wikimedia.org/event/wikidata-data-reuse-days-2022/) (my first Wikimedia event). It is strongly recommended that "reusers" of Wikidata and other Wikimedia projects directly cite where they get their information from. Wikidata is CC0, so you technically don't have to, but in conversations with Wikidata employees, editors and the general community they very much like to get recognition where possible. This is also why we have the "With data from Wikimedia" along with the Wikidata logo and Wikipedia logo on the screens for the App Store.

Legally speaking we'd fall within the section that you pointed out, @wkyoshida, but beyond that is a strong suggestion that we include the icon. There are "Powered by Wikidata" logos on the [data access page](https://www.wikidata.org/wiki/Wikidata:Data_access), but then I'm not a big fan of those and chose to just go with the logos instead 🙃

Reply:
Awesome! :rocket: I was quite certain there must have been some prior conversation/recommendation from the community and was more just curious on how that had played out. Thanks for the comment! Exactly what I was wondering :+1: